core/src/main/scala/org/allenai/common/StringImplicits.scala

```diff
@@ -1,115 +0,0 @@
-package org.allenai.common
```

Comment:
That's not enough to be true source-code compatible. The old `StringImplicits` had members that could have been accessed, and the names of those members aren't the same as they are in `StringUtils`.

However, this way at least you'll get a useful warning telling you why it breaks. If you think that's good enough, I'll send a code review with that change.